openedx/core/djangoapps/oauth_dispatch/tests/test_views.py

```diff
@@ -242,33 +260,39 @@ def test_access_token_metrics_for_bad_request(self, mock_set_custom_metric):
         mock_set_custom_metric.assert_has_calls(expected_calls, any_order=True)
 
     @ddt.data(
-        (False, True, settings.DEFAULT_JWT_ISSUER),
-        (True, False, settings.RESTRICTED_APPLICATION_JWT_ISSUER),
+        (False, True, False),
+        (True, False, True),
     )
     @ddt.unpack
     def test_restricted_jwt_access_token(self, enforce_jwt_scopes_enabled, expiration_expected,
-                                         jwt_issuer_expected):
+                                         asymmetric_key_expected):
```

Comment:
Why is this a parameter?  Should there just be a line below: `asymmetric_key_expected = enforce_jwt_scopes_enabled`?  Or is this inaccurate?

Reply:
yup. updated.